sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -208,6 +210,9 @@ def recognize_entities(  # type: ignore
             is not specified, the API will default to the latest, non-preview version.
         :keyword bool show_stats: If set to true, response will contain document
             level statistics in the `statistics` field of the document-level response.
+        :keyword str string_index_type: Specifies the method used to interpret string offsets.
```

Comment:
I would be. more clear here and say something like this. def not 100% recommedingn this docstring, justs want. it be more clear that you really should'nt touch this unlesss you want to override default python behavior
```
:keyword str string_index_type: Specifies the method used to interpret string offsets. `UnicodeCodePoint`, the Python encoding is the default. To override the Python default, you can also pass inn `Utf16CodePoint` or `TextElements_v8`. See ... for additional innformation
```

Reply:
Ok, I updated all of the docstrings with what you shared above.